### ISSUE TITLE
Impossible to debug, single_gpu code paths are broken

### ISSUE THREAD

**Issue by aviadmx** [zhiqi-li/Panoptic-SegFormer#9]
It seems that the multi gpu training and eval works great, however, while trying to debug you're opt for using a single gpu.  
In that case the code breaks in several parts during the evaluation of the validation set.  
Any chance for a hotfix? :)

To reproduce, try to run the code from PyCharm in debug mode while there's only one GPU available.

**Comment by BarCodeReader** [comment]
I have a similar issue but not this repo. debug MMdet-related code is almost impossible, they encapsulate everything and it is not worth the time to debug, you may use that time to create a pure Pytorch model already :)
